hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieMetricsPrometheusConfig.java

```diff
@@ -70,7 +70,7 @@
       .key(PUSHGATEWAY_PREFIX + ".random.job.name.suffix")
       .defaultValue(true)
       .sinceVersion("0.6.0")
-      .withDocumentation("");
+      .withDocumentation("Pushgateway job name need a random suffix, default true.");
```

Comment:
“Whether the pushgateway name need a random suffix , default true” 
might be better ?

Reply:
done